bundler-audit.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |gem|
   gem.files = if gemspec['files']
                 glob[gemspec['files']]
               else
-                `git ls-files`.split($/)
+                Dir.glob(%w[bin/**/* lib/**/* bundler-audit.gemspec ChangeLog.md COPYING.txt gemspec.yml README.md])
```

Comment:
I went with the `Dir.glob` approach out of personal preference, but if `git ls-files` is preferred, I can change this to use that instead.

Reply:
Yes `git ls-files` is preferred as that prevents non-git-tracked files from sneaking into releases.